### ISSUE TITLE
Translations for 2.1.0

### ISSUE THREAD

**Issue by splittingred** [modxbot/migrate#4216]
splittingred created Redmine issue ID 4216

Translations are needed for Revolution 2.1.0. There are some slight changes to strings, and a few new settings and strings. Please attach all translations here, in a single zip file, named "language-2.1.0-pl.zip", where language is the IANA code of your translation.

Alternatively (and preferred!), you can send a pull request on GitHub to the revolution repository to send the files. The core team will review them and then commit them to Git. Please sign a CLA via https://rightsignature.com/documents/34649BJ8R3X8Y95EDLEWIT Translations cannot be accepted without a CLA

We will accept translations until 2PM Central Time (GMT -0600), April 8, 2011.

_Please ensure all contributions use UNIX-style line-endings only._

**Comment by charliez** [comment]
charliez submitted:

Here is the Spanish (es-mx) Translations of Revo 2.1-RC1

**Comment by splittingred** [comment]
splittingred submitted:

Updated Spanish translation. Thanks Carlos!

**Comment by Pont** [comment]
pont submitted:

Swedish translation for 2.1.0

**Comment by splittingred** [comment]
splittingred submitted:

Swedish translation updated: https://github.com/modxcms/revolution/commit/9a4a13d2d34c9d757fb27c412b047ef41aff9940

**Comment by bertoost** [comment]
bertoost submitted:

@Shaun,
Could translators like me have an email when new translations are requested? Because I do not look on the bugtracker every day I saw the request about translations for 2.1.0 today. I will do my best again to update the Dutch translation before the release.. but I missed 5 days for this :-(.. maybe it's a good thing to keep tracking the translators in a special newsletter-group and send an email if an update is requested?

**Comment by Pont** [comment]
pont submitted:

Updated swedish core default.inc.php

**Comment by splittingred** [comment]
splittingred submitted:

Updated! Thanks.

Bert, we're looking into more formally developing the Translation teams, and giving more feedback and notifications for them. We'll let you know.

**Comment by onepack** [comment]
onepack submitted:

Hi Shaun,

I would like to contribute in this but have no idea where to get the latest pl files that need to be translated and what rules apply. How do I know that the translations I'm working on are not already done/ in progress by some others. Is there a way to do this in a different way. Have the files online to be updated online by users that get access to do so by modx so the modx team can easily track all the changes etc.

**Comment by Pont** [comment]
pont submitted:

Swedish files updated with the latest additions

**Comment by modxbot** [comment]
enigmatic_user submitted:

onepack onepack wrote:

> I would like to contribute in this but have no idea where to get the latest pl files that need to be translated

The latest version of MODX Revolution can always be found at GitHub (https://github.com/modxcms/revolution). You'll find more information on this at http://rtfm.modx.com/display/community/Using+Git+and+GitHub and its subpages.

Nomally the develop branch is the right one; when a new version is due, there's a release branch for that version (at the moment it's release-2.1.0).

**Comment by pandro** [comment]
pandro submitted:

Here is the Italian (it) Translations of Revo 2.1

**Comment by splittingred** [comment]
splittingred submitted:

Updated. Thanks guys!

**Comment by bertoost** [comment]
bertoost submitted:

Here the Dutch translation

**Comment by splittingred** [comment]
splittingred submitted:

Updated. Thanks Bert.

**Comment by pandro** [comment]
pandro submitted:

Here the Italian translation updated for RC3, 
thanks!
Alessandro

**Comment by splittingred** [comment]
splittingred submitted:

Updated.

**Comment by pandro** [comment]
pandro submitted:

Here is 2.1 rc4, 
thanks,
Alessandro

**Comment by Pont** [comment]
pont submitted:

Swedish language updates for rc4

**Comment by splittingred** [comment]
splittingred submitted:

Updated. Thanks guys.

**Comment by gadamiak** [comment]
gadamiak submitted:

I've started work on Polish translation.

I noticed there are 171 duplicate entries, which get overwritten when included in files with names starting on letters from the end of the alphabet. Is there any other purpose to using multiple files other than keeping them organized? I guess not, so what's the key: manager template? A cleanup is required anyway.

I've also found some entries to be unused or replaced with others, e.g. @recent_docs@ is used instead of @activity_title@, @info@ is used in place of @yourinfo_title@. This would require a cleanup too.
